scripts/contentScript.js

```diff
@@ -382,15 +382,24 @@ async function FilesInPrBitbucket(response) {
 
 const githubHunkHighlight = async (apiResponses) => {
 
-	// Todo : optomization needed to not highlight next deleted line space if not present in response. 
+	// Todo : optomization needed to not highlight next deleted line space if not present in response.
+	console.log("api responses = ", apiResponses);
 	let getFileName = Array.from(document.querySelectorAll('div[data-tagsearch-path]'));
 	getFileName.forEach(async (item) => {
 		let FileContent = item.getAttribute('data-tagsearch-path');
 		if (FileContent) {
 
 			let matchEncrypted = await sha256(FileContent);
-			let foundFile = apiResponses.some(item => item.filepath === matchEncrypted);
-
```

Comment:
sorry tapish, actually I did it right but because of previous stashing I forgot to remove .some and place ".find" instead of it.

it'll be more efficient in case of large array of object.